tests/support_server/wait_support_server.pm

```diff
@@ -7,8 +7,8 @@
 # notice and this notice are preserved.  This file is offered as-is,
 # without any warranty.
 
-# Summary: Start HA support server and check network connectivity
-# Maintainer: Loic Devulder <ldevulder@suse.com>
+# Summary: Start support server and check network connectivity
```

Comment:
My summary seems a little bit old, as there is no network connectivity check in it...

A better summary could be simply: "Wait for support server".